certbot-apache/certbot_apache/constants.py

```diff
@@ -80,7 +80,7 @@
     "red hat enterprise linux server": CLI_DEFAULTS_CENTOS,
     "rhel": CLI_DEFAULTS_CENTOS,
     "amazon": CLI_DEFAULTS_CENTOS,
-    "gentoo base system": CLI_DEFAULTS_GENTOO,
```

Comment:
Do not delete this line, just add the new one for the systemd /etc/os-release string.